pyroma/ratings.py

```diff
@@ -392,6 +392,16 @@ def message(self):
         return "You should have three or more owners of the project on PyPI."
 
 
+class TestSuitDeprecation(BaseTest):
+    weight = 50
+
+    def test(self, data):
+        return "test_suite" not in data
```

Comment:
FYI, this check doesn't actually work, since as mentioned on #72 , this is a Setuptools-specific option (under the `[options]` section in `setup.cfg`, or the `[tool.setuptools]` table in `pyproject.toml`), not anything in the standardized core metadata like everything else you check here.

Reply:
Well, it only works if you use the fallback, so it's a bit pointless, you are right, because that already gets a much more important warning.

Reply:
Yes, right, it only works when the metadata is fetched by  patching setuptools. So it's useful in Pyroma 3 (which this was) but for Pyroma 4 it's pointless, true. So I can already remove it again. :-)